app/src/components/ExplorePage/ExplorePinnedTopics.vue

```diff
@@ -0,0 +1,97 @@
+<script setup lang="ts">
+import { watch } from "vue";
+import {
+    type ContentDto,
+    DocType,
+    type Uuid,
+    db,
+    useDexieLiveQueryWithDeps,
+} from "luminary-shared";
+import { appLanguageIdAsRef } from "@/globalConfig";
+import { contentByTopic } from "@/components/ExplorePage/contentByTopic";
+import HorizontalContentTileCollection from "@/components/content/HorizontalContentTileCollection.vue";
+import IgnorePagePadding from "../IgnorePagePadding.vue";
+// import VerticalTagViewer from "../tags/VerticalTagViewer.vue";
+
+const pinnedTopics = useDexieLiveQueryWithDeps(
+    appLanguageIdAsRef,
+    (appLanguageId: Uuid) =>
+        db.docs
+            .where({
+                type: DocType.Content,
+                language: appLanguageId,
+                status: "published",
+                parentPinned: 1, // 1 = true
+                // parentTagType: TagType.Topic,
+            })
+            .filter((c) => {
+                const content = c as ContentDto;
+                if (!content.publishDate) return false;
+                if (content.publishDate > Date.now()) return false;
+                if (content.expiryDate && content.expiryDate < Date.now()) return false;
+                return true;
+            })
+            .toArray() as unknown as Promise<ContentDto[]>,
+    { initialValue: await db.getQueryCache<ContentDto[]>("explore_pinnedTopics") },
+);
+
+watch(pinnedTopics, async (value) => {
+    db.setQueryCache<ContentDto[]>("explore_pinnedTopics", value);
+});
+
+const pinnedTopicContent = useDexieLiveQueryWithDeps(
+    [appLanguageIdAsRef, pinnedTopics],
+    ([appLanguageId, pinnedTopics]: [Uuid, ContentDto[]]) =>
+        db.docs
+            .where({
+                type: DocType.Content,
+                language: appLanguageId,
+                status: "published",
+            })
```

Comment:
This is a very heavy query, which can be improved when #463 is implemented. Please create a separate ticket to redo this query?